src/app/upgrade.h

```diff
@@ -210,4 +215,29 @@ bool upgrade(bool ask = true)
     return true;
 }
 
+
+#ifdef Q_OS_MAC
+bool copyPlistToIni(const char *application)
```

Comment:
And probably this shold be called `migratePlistToIni()`
Also why take a `const char*` when you can do `const QString &application` ?